adafruit_character_lcd/character_lcd.py

```diff
@@ -448,33 +463,33 @@ def text_direction(self):
         return self._direction
 
     @text_direction.setter
-    def text_direction(self, direction):
+    def text_direction(self, direction: int) -> None:
         self._direction = direction
         if direction == self.LEFT_TO_RIGHT:
             self._left_to_right()
         elif direction == self.RIGHT_TO_LEFT:
             self._right_to_left()
 
-    def _left_to_right(self):
+    def _left_to_right(self) -> None:
         # Displays text from left to right on the LCD.
         self.displaymode |= _LCD_ENTRYLEFT
         self._write8(_LCD_ENTRYMODESET | self.displaymode)
 
-    def _right_to_left(self):
+    def _right_to_left(self) -> None:
         # Displays text from right to left on the LCD.
         self.displaymode &= ~_LCD_ENTRYLEFT
         self._write8(_LCD_ENTRYMODESET | self.displaymode)
 
-    def create_char(self, location, pattern):
+    def create_char(self, location: int, pattern: Sequence[int]) -> None:
         """
         Fill one of the first 8 CGRAM locations with custom characters.
         The location parameter should be between 0 and 7 and pattern should
         provide an array of 8 bytes containing the pattern. E.g. you can easily
         design your custom character at http://www.quinapalus.com/hd44780udg.html
         To show your custom character use, for example, ``lcd.message = "\x01"``
 
-        :param location: integer in range(8) to store the created character
-        :param ~bytes pattern: len(8) describes created character
+        :param int location: Integer in range(8) to store the created character.
+        :param Sequence pattern: len(8) describes created character.
```

Comment:
Since `Sequence` is a generic, you could actually use `Sequence[int]`!

Reply:
Got it, thanks!